docs/ADR-72-ab-testing.md

```diff
@@ -0,0 +1,153 @@
+# A/B testing
+
+## Context and Problem Statement
+
+Given the amount of new features we need a way to measure the impact that each one has on the use of our products and A/B testing is the way we use to decide between two or more versions of a new feature and/or compared with the current one. This ADR describe how we measure A/B testings in our products.
+
+## Main objectives
+
+- sanitize metrics
+- allow data team to develop, monitor and improve dashboards and tools decoupled from the implementation on each app
+- allow any team to implement a/b testings on their environment without dependencies with other teams
+- leverage over already implemented and tested tools
+
+## Current implementation
+
+To setup an A/B testing we use Feature Flags as assignation tool and Segment as measurement tool.
+
+### Feature Flags: Setting up a new A/B testing
+
+On this standard an A/B is defined as a feature flag that ends with `_variant` with each of the possible variants defined as variant and the activation defined as your application required.
+
+> Please note that if you need to test a new variant against the current one it is preferable to use a variant with two options (`enabled` and `disabled`) instead of the activation because otherwise you won't be able to differentiate (and measure) when your a/b is active or just should show the current version.
+
+#### Creating a new flag
+
+Just create a new feature that ends `_variant`
+
+![Create new flag](./resources/ADR-70/001-create-new-feature.png)
+
+#### Defining your activation
+
+You can define your activation as your project needs, but in most of the cases you will need one of this two options:
+
+- activation by host: `applicationHostname`
+- activation rollout session: `gradualRolloutSessionId`
+
+`applicationHostname` allows you to keep your test active on any testing environment and when its active on production
+
+![Activation by host](./resources/ADR-70/002-activation-by-host.png)
+
+If you also add `gradualRolloutSessionId` you can rollout your test gradually
+
+![Activation rollout](./resources/ADR-70/003-activation-rollout.png)
+
+#### Defining your variants
+
+In the `VARIANT` tab you can define as many cases as you need and control the percentage of users that will get each variant.
+
+![Add variant](./resources/ADR-70/004-add-variant.png)
+
+![List variants](./resources/ADR-70/005-varian-list.png)
+
+> Please note that if you define and activation with rollout and `enabled`/`disabled` variants you will split your users as follow:
+>
+> - assuming total: `1.000` users
+> - activation rollout (50%): `500` users
```

Comment:
Why do we do this? Why not 500 users on A and 500 on B?

Reply:
Just as an example to watch out, let me add something to be more clear